build.gradle.kts

```diff
@@ -93,6 +94,15 @@ tasks {
 	}
 }
 
+sonar {
+	properties {
+		property("sonar.projectKey", "MockBukkit_MockBukkit")
+		property("sonar.organization", "mockbukkit")
+		property("sonar.projectVersion", "v${property("paper.api.full-version")}")
```

Comment:
Should this use `paper.api.version` (e.g. `1.20`) instead of the full-version (`1.20.2-R0.1-SNAPSHOT`)?